docs/ui.md

```diff
@@ -2,30 +2,32 @@
 title: Build the Connection Dialog with Connection Dialog v1
```

Comment:
Probably a good idea to change the title of this page to match what it is in the index, same with the Connection Dialog v2 page